jira/plugin/core/src/main/java/org/overbaard/jira/impl/BoardConfigurationManagerImpl.java

```diff
@@ -158,6 +158,16 @@ public BoardConfig getBoardConfigForBoardDisplay(ApplicationUser user, final Str
 
     @Override
     public BoardConfig getBoardConfig(final String code) {
+        return getBoardConfig(code, false);
```

Comment:
Shouldn't this be `return getBoardConfig(code, true);` to force reread from DB?

Reply:
I don't think so since a lot of the time we want to use the cached information. I see some other problems though so I'll push with new fixes.